api/tests/python/end_points/payments/test_payments.py

```diff
@@ -88,123 +58,143 @@ def setup_draft_nr(client):
 
+@pytest.mark.skip
+def verify_payment_payload(payload):
     assert isinstance(payload.get('id'), int) is True
-    assert isinstance(payload.get('invoices'), list) is True
+    assert isinstance(payload.get('nrId'), int) is True
+    assert isinstance(payload.get('payment'), dict) is True
+    assert isinstance(payload.get('sbcPayment'), dict) is True
+    assert isinstance(payload.get('statusCode'), str) is True
+    assert isinstance(payload.get('completionDate'), str) is True
+    assert isinstance(payload.get('token'), str) is True
 
 
-def test_create_payment(client, jwt, app):
-    draft_nr = setup_draft_nr(client)
+@pytest.mark.skip
+def execute_calculate_regular_fees(client):
+    """
+    1) Get the current the fees.
+    :param client:
+    :return:
+    """
+    headers = get_test_headers()
 
-    nr_num = draft_nr.get('nrNum')
-    request_uri = API_BASE_URI + nr_num
+    request_uri = API_BASE_URI + 'fees'
 
+    # Test regular submission
     path = request_uri
-    body = json.dumps(create_payment_request)
+    body = json.dumps({
+        'corp_type': 'NRO',
+        'filing_type_code': 'NM620',
+        'jurisdiction': 'BC',
+        'date': '',
+        'priority': ''
+    })
     log_request_path(path)
 
-    response = client.post(path, json=body)
+    response = client.post(path, data=body, headers=headers)
+
+    assert response.status_code == 200
+
     payload = json.loads(response.data)
+    verify_fees_payload(payload)
 
-    assert isinstance(payload.get('id'), int) is True
-    assert isinstance(payload.get('invoices'), list) is True
-    assert payload.get('updated_by') is None
-    assert payload.get('updated_on') is None
+    assert payload.get('filingTypeCode') == 'NM620'
 
+    return payload
 
-def test_update_payment(client, jwt, app):
-    draft_nr = setup_draft_nr(client)
 
-    nr_num = draft_nr.get('nrNum')
-    request_uri = API_BASE_URI + nr_num
+@pytest.mark.skip
+def execute_calculate_upgrade_fees(client):
+    """
+    1) Get the current the fees.
+    :param client:
+    :return:
+    """
+    headers = get_test_headers()
 
-    payment_id = 'abcd153'
-    request_uri = API_BASE_URI + payment_id
+    request_uri = API_BASE_URI + 'fees'
 
+    # Test regular submission
     path = request_uri
-    body = json.dumps(create_payment_request)
+    body = json.dumps({
+        'corp_type': 'NRO',
+        'filing_type_code': 'NM606',
+        'jurisdiction': 'BC',
+        'date': '',
+        'priority': ''
+    })
     log_request_path(path)
 
-    response = client.put(path, json=body)
-    payload = json.loads(response.data)
+    response = client.post(path, data=body, headers=headers)
 
-    assert isinstance(payload.get('id'), int) is True
-    assert isinstance(payload.get('invoices'), list) is True
-    # TODO: Test update fields
-    # assert payload.get('updated_by') is not None
-    # assert payload.get('updated_on') is not None
+    assert response.status_code == 200
```

Comment:
201?

Reply:
Returning 200 simply because there's actually nothing created despite it being a post, the fees are kinda the fees it's a calculation result as opposed to a resource. Just personal preference or whatever...